js/src/reports/metric-number.js

```diff
@@ -48,27 +49,50 @@ const MetricNumber = ( {
 		const formatFn = isCurrency ? formatAmount : formatNumber;
 
 		return {
-			value: formatFn( value ),
+			value:
+				value === undefined
+					? __( 'Unavailable', 'google-listings-and-ads' )
+					: formatFn( value ),
 			prevValue: formatFn( prevValue ),
 		};
 	}, [ isCurrency, value, prevValue, formatNumber, formatAmount ] );
 
 	let markedLabel = label;
+	const infos = [];
 
 	// Until ~Q4 2021, metrics for all programs, may lack data for free listings.
-	if ( missingFreeListingsData ) {
-		const infoText = __(
-			'This data is available for paid campaigns only.',
-			'google-listings-and-ads'
+	// And Free Listings API may not respond with data.
+	if ( missingFreeListingsData !== MISSING_FREE_LISTINGS_DATA.NONE ) {
+		infos.push(
+			__(
+				'This data is currently available for paid campaigns only.',
+				'google-listings-and-ads'
+			)
 		);
+	}
+	if ( missingFreeListingsData === MISSING_FREE_LISTINGS_DATA.FOR_REQUEST ) {
```

Comment:
💅 Line 65 uses `!==`, and line 73 uses `===`. Sometimes I'm a bit worried about situation like this, where we might accidentally read it wrongly and thought both are `!==` or both are `===`. Personally, I _might_ write line 65 like the following: 

```js
// instead of: if ( missingFreeListingsData !== MISSING_FREE_LISTINGS_DATA.NONE ) {
if ( [ MISSING_FREE_LISTINGS_DATA.FOR_METRIC, MISSING_FREE_LISTINGS_DATA.FOR_REQUEST ].includes( missingFreeListingsData ) ) {
```

However, the above style is arguable too. Imagine that we have more MISSING_FREE_LISTINGS_DATA in the future, we might need to add them into the array, which can be problematic and easily missed too. 

Anyway, I guess it's probably fine to leave what you have here as is. 😄  Just voicing things out from my head.